### DISCUSSION TITLE
Is pyFaidx compatible with 3.12?

### DISCUSSION THREAD

**Discussion by duartemolha** [mdshw5/pyfaidx#219] [Q&A]
I see in the setup that you accept any python above 3.7.. but i have instaled it in 3.12 and have been getting errors whilst it works fine in 3.11.

**Comment by mdshw5** [comment]
I've just added 3.12 to the test environment and see that all tests pass under both linux and macOS: https://github.com/mdshw5/pyfaidx/actions/runs/8834316745/job/24255677328. Can you share the specific error that you're getting under 3.12? If so I would be glad to take a look.
